Hunting Queries/GitLab/GitLab_LocalAuthNoMFA

```diff
@@ -0,0 +1,34 @@
+id: e0b45487-5c79-482d-8ac0-695de8c031af
+name: GitLab - Local Auth - No MFA 
+description: |
+  'This query checks GitLab Audit Logs to see if a user authenticated without MFA. Ot might mean that MFA was disabled for the GitLab server or that an external authentication provider was bypassed.
+ This rule focuses on 'admin' privileges but the parameter can be adapted to also include all users.'
+severity: Medium
+requiredDataConnectors:
+   - connectorId: Syslog
+    dataTypes:
+      - Syslog
+queryFrequency: 1h
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Credential Access
+relevantTechniques:
+  - T1110
+query: |
+
+let isAdmin = true;
```

Comment:
Can you please double check if the query runs correctly without errors and gives expected results?

Reply:
Yes double-checked.